app/main/views/marketplace.py

```diff
@@ -75,14 +75,23 @@ def terms_and_conditions():
 def get_brief_by_id(framework_framework, brief_id):
     briefs = data_api_client.get_brief(brief_id)
     brief = briefs.get('briefs')
+    brief_responses = data_api_client.find_brief_responses(
+        brief_id=brief_id,
+        status='draft,submitted'
+    ).get("briefResponses")
+    started_brief_responses_count = len([response for response in brief_responses if "submittedAt" not in response])
+    completed_brief_responses_count = len([response for response in brief_responses if "submittedAt" in response])
+
     if brief['status'] not in ['live', 'closed', 'withdrawn'] or brief['frameworkFramework'] != framework_framework:
         abort(404, "Opportunity '{}' can not be found".format(brief_id))
 
     if getattr(current_user, "supplier_id", None) is None:
         # user unauthenticated or not a supplier
-        brief_responses = None
+        supplier_brief_response = None
     else:
-        brief_responses = data_api_client.find_brief_responses(brief_id, current_user.supplier_id)["briefResponses"]
+        supplier_brief_response = [
+            response for response in brief_responses if response["supplierId"] == current_user.supplier_id
```

Comment:
I can't see where this brief response itself is ever used; `supplier_brief_response` only seems to be used as a boolean. We should probably turn it into one and call it something like `supplier_has_responded_to_brief` so that it's clearer what this is being used for. I know you've not written this bit from scratch, but since you're touching it we might as well reduce it to the simplest case.